lib/services/cdnManagement/package.json

```diff
@@ -6,7 +6,7 @@
     "Zavery, Amar <amzavery@microsoft.com>",
     "Wan, Qinyuan <qinwa@microsoft.com>"
   ],
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
The previous change updated from 0.2.1 to 1.0.0, but the example in README is not changed, so the major version change already happened,